### ISSUE TITLE
Add support for postgres enums

### ISSUE THREAD

**Issue by Zhurik** [KarnerTh/mermerd#15]
We have a lot of enums in our production database, so I want to show them in ERD diagram. Right now they appear simply as USER-DEFINED. How do you look at fetching data about enums from DB and fitting those values into the comment section of describing tables ([here](https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram?id=attribute-keys-and-comments))
How it works right now:

```mermaid
erDiagram
  table_name {
          USER-DEFINED best_column
  }
```

How I propose it to look like:

```mermaid
erDiagram
  table_name {
          myFavouriteEnumType best_column "MyFavouriteEnumVal1, MyFavouriteEnumVal2, MyFavouriteEnumVal3"
  }
```

If you're okey with this proposal I can make PR later

**Comment by KarnerTh** [comment]
I have a similar idea in mind, but have not started implementing it yet.
It should play well with the existing feature to display the constraint name in the column and should be an opt-in.

I have it on my personal todo list, but if you like to make a suggestion feel free to create a PR :)

**Comment by Zhurik** [comment]
I created PR for this https://github.com/KarnerTh/mermerd/pull/17
Yes, it does not hoave proper tests for it, because it needs separate sql for creating some columns in Postgres db

**Comment by KarnerTh** [comment]
Feature will be released with version 0.5.0
